Add threading info (#1446)
fluent/fluent-bit-docs: pipeline/inputs/nginx.md

### Configuration with NGINX Plus REST API

> A much more powerful and flexible metrics API is available with NGINX Plus. A path needs to be configured
> in NGINX Plus first.

Reviewer: ```suggestion
Another metrics API is available with NGINX Plus. You must first configure a path in
NGINX Plus.
```
